src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,8 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             String routeId = null;
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
+            } else if (informed.hasTrip() && informed.getTrip().hasRouteId())  {
+                routeId = informed.getTrip().getRouteId();
```

Comment:
It is not appropriate to set the overall route_id for an Alert using this trip descriptor alert. The [docs on gtfs.org](http://gtfs.org/reference/realtime/v2/#message-tripdescriptor) explicitly state this:

> TripDescriptor.route_id cannot be used within an Alert EntitySelector to specify a route-wide alert that affects all trips for a route - use EntitySelector.route_id instead.